config/jobs/kubernetes/sig-node/dynamic-resource-allocation.yaml

```diff
@@ -0,0 +1,39 @@
+periodics:
+  # This jobs runs e2e.test with a focus on tests for the Dynamic Resource Allocation feature (currently alpha)
+  # on a kind cluster with containerd updated to a version with CDI support.
+  - name: ci-kind-dra
+    interval: 6h
+    annotations:
+      testgrid-dashboards: sig-node-dynamic-resource-allocation
+      testgrid-tab-name: ci-kind-dra
```

Comment:
I'm a bit unsure where I should add jobs to the testgrid: here or in [config/testgrids/kubernetes/sig-node/config.yaml](https://github.com/kubernetes/test-infra/pull/27997/files#diff-63006db0bad1cc1272c9a042e7864ff01dbfe8b12c866f2676a3eee359c8c028)? I'm currently doing both.

Reply:
That led to a config check failure about double usage of "cikinddra". I'm removing it here.

Reply:
Still some conflict. Let's do it all here, including the alert email...